jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -30,11 +28,20 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     assert mm.results_label_overwrite is True
 
     result = dc[1].get_object(cls=CCDData)
-    assert result.shape == (2, 4)  # Cube shape is (2, 2, 4)
+    assert result.shape == (4, 2)  # Cube shape is (2, 2, 4)
```

Comment:
But in changing from using `app.add_data` to `cubeviz_helper.load_data`, the moment map shape changed! @rosteen , since you refactored how Cubeviz swaps dimensions around, does this make sense to you?